README.md

```diff
@@ -14,31 +14,34 @@ Here is an example usage that would be deployed in multiple instances
 package main
 
 import (
+	"fmt"
 	"time"
 
 	"github.com/go-co-op/gocron"
-	redislock "github.com/go-co-op/gocron-redis-lock"
 	"github.com/redis/go-redis/v9"
+
+	redislock "github.com/go-co-op/gocron-redis-lock"
 )
 
 func main() {
 	redisOptions := &redis.Options{
 		Addr: "localhost:6379",
 	}
 	redisClient := redis.NewClient(redisOptions)
-	locker, err := redislock.NewRedisLocker(redisClient)
+	locker, err := redislock.NewRedisLocker(redisClient, redislock.WithTries(1))
 	if err != nil {
-		// handle the error
+		panic(err)
```

Comment:
small thing - i don't like panics in examples because it's not something i'd ever really encourage a dev to do
```suggestion
		// handle the error
```